docs/kh2/file/type/mset.md

```diff
@@ -0,0 +1,81 @@
+# [Kingdom Hearts II](../../index.md) - MSET (Moveset)
+
+The MSET files contain animations and the effects related to them (I-frames, hitboxes, effects...)
+It is a [BAR file](bar.md) that contains a list of "slots". Each slot may or may not have a "motion".
+The various actions in the game point to a specific motion by a slot position in this list, thus many of the motions are dummies, since the characters only use some of them.
+Eg: For Sora, slot 0 is idle, slot 4 is walking and slot 604 is the basic attack.
+
+Each motion is a BAR file with 2 entries, the model animation and the effects.
+
+## Animation structure
+
+???
+
+## Effects structure
+
+| Amount | Description |
+|--------|---------------|
+| 1 	 |  Header
+| X 	 |  Type A effect (Happens during X frames)
+| X 	 |  Type B effect (Triggered on a specific frame)
+
+### Effects header
+
+| Offset | Variable Type | Description |
+|--------|---------------|-------------|
+| 0 	 | byte | Type A effect count
+| 1 	 | byte | Type B effect count
+| 2 	 | short | Start offset of the type B effects
+
+### Type A effect
+
+| Offset | Variable Type | Description |
+|--------|---------------|-------------|
+| 0 	 | short | Start frame
+| 2 	 | short | End frame
+| 4 	 | byte | Effect A ID
```

Comment:
Just to polish this doc by a bit: could you add a link to the enumerations below the structures? For example Effect A ID should have a link to #effect-a-list

Reply:
Added multiple links for easier navigation.